generic/tests/ioquit.py

```diff
@@ -14,6 +18,7 @@ def run(test, params, env):
     2) Add IO workload for guest OS
     3) Sleep for a random time
     4) Kill the VM
+    5) Run image check, and if cluster leaks exist, try to repair them
```

Comment:
"Check the image and the expected result is no errors are found except cluster leaks."